test/e2e/cp_test.go

```diff
@@ -91,6 +91,49 @@ var _ = Describe("Podman cp", func() {
 		Expect(roundtripContent).To(Equal(originalContent))
 	})
 
+	// Copy a file to the container, then back to the host in --pid=host
+	It("podman cp --pid=host file", func() {
```

Comment:
Apologies for noticing the PR just now:  Would you consider moving the test logic to the system tests?  When rewriting the copy backend, I made sure to have a good coverage in the system tests since they're used in gating and also by QE.